moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -34,22 +34,21 @@
  *********************************************************************/
 
 /*
- * IKFast Plugin Template for moveit
+ * IKFast Kinematics Solver Plugin for MoveIt! wrapping an ikfast solver from OpenRAVE.
  *
- * AUTO-GENERATED by create_ikfast_moveit_plugin.py in arm_kinematics_tools
- * You should run create_ikfast_moveit_plugin.py to generate your own plugin.
- *
- * Creates a kinematics plugin using the output of IKFast from OpenRAVE.
- * This plugin and the move_group node can be used as a general
- * kinematics service, from within the MoveIt! planning environment, or in
- * your own ROS node.
```

Comment:
please restore this documentation

Reply:
This documentation is redundant and misleading. That's why I shortened it.

> You should run create_ikfast_moveit_plugin.py to generate your own plugin.

That's already stated by the line `AUTO-GENERATED by create_ikfast_moveit_plugin.py`.

> Creates a kinematics plugin using the output of IKFast from OpenRAVE.

Neither the template nor the generated plugin _create_ a kinematics plugin. The file _is the source_ for the IK plugin.

>  This plugin and the move_group node can be used as a general kinematics service, from within the MoveIt! planning environment, or in your own ROS node.

Using a move_group service is not the standard way to use a IK plugin anymore. As a plugin it can be used directly.

Reply:
I think the first two sentences are much more explicit than the very terse one-liner you've settled on. And the second one is mostly correct, but I see your point about it being possibly misinterpreted. instead of removing it, please tweak the wording:

> This file, combined with the output of IKFast from OpenRAVE, is used to create a MoveIt! kinematics plugin